scripts/content.js

```diff
@@ -35,8 +35,8 @@ fetch(proxyurl + url) // https://cors-anywhere.herokuapp.com/https://example.com
           linksList.forEach(function(product) {
             const node = product.childNodes[0];
             const label = node.getAttribute("aria-label");
-            let webbrand = label.split(" ")[0];
-            if(brand[0] === webbrand){
+            let webbrand = label.toLowerCase();
```

Comment:
this is a very confusing variable name for two reasons:

- can get confused with your `webbrands`
- is no longer a brand, is the entire label

maybe change to `productNodeText`